<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeAutowidget</class>
 <widget class="QWidget" name="ModeAutowidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VL_menuTransaction">
    <item>
     <layout class="QHBoxLayout" name="HL_infoMontant">
      <item>
       <widget class="QLabel" name="label_montantBase">
        <property name="text">
         <string>MontantB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_montantContrepartie">
        <property name="text">
         <string>MontantC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_broker">
        <property name="text">
         <string>Broker</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_strategie_2">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Strategie</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_timer">
      <item>
       <property name="text">
        <string>Pause</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Avancer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reculer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DÃ©filer</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="HL_infoOpen">
      <item>
       <widget class="QLabel" name="label_date">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_openPrice">
        <property name="text">
         <string>OpenPrice</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_montant"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="HL_achatvente">
      <item>
       <widget class="QPushButton" name="pushButton_achat">
        <property name="text">
         <string>Achat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_vente">
        <property name="text">
         <string>Vente</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_annulerTransaction">
      <property name="text">
       <string>Annuler transaction</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
